Fabric.Identity.API/scripts/azuresettings.config

```diff
@@ -19,7 +19,9 @@
         <variable name="" alias="" />
       </allowedTenants>
       <!-- The name of the secret to be created for the azure application registration -->
+      <secretName>
```

Comment:
Do the registration scripts need to change because of this as well?

Reply:
added change to registration script